sklearn/covariance/_elliptic_envelope.py

```diff
@@ -171,6 +178,7 @@ def fit(self, X, y=None):
         self : object
             Returns the instance itself.
         """
+        self._validate_params()
```

Comment:
No need to call the validation instead add the following comment

```python
# `_validate_params` is called in `MinCovDet`
```

Reply:
done